MapboxVisionAR/VIsionARManager/VisionARManagerDelegate.swift

```diff
@@ -31,9 +31,14 @@ public protocol VisionARManagerDelegate: AnyObject {
     func visionARManager(_ visionARManager: VisionARManager, didUpdateARLaneCutoff: Float)
 
     /**
-     AR fences were updated
+     Tells the delegate that AR fences were updated.
      */
     func visionARManager(_ visionARManager: VisionARManager, didUpdateARFences arFences: [ARFence])
+
+    /**
+     Tells the delegate that route was updated.
+     */
+    func visionARManager(_ visionARManager: VisionARManager, didUpdateRoute route: Route)
```

Comment:
Should we implement `onRouteUpdated:` in `VisionARManager.swift`?

Reply:
Sure, I missed it. Thanks